metadata-ingestion/schedule_docs/airflow.md

```diff
@@ -0,0 +1,11 @@
+# Using Airflow
+
+If you are using Apache Airflow for your scheduling then you might want to use it for scheduling your ingestion receipes. To do that you can follow these steps
+- Create a receipe file e.g. `recipe.yml`
+- Ensure [DataHub CLI](../../docs/cli.md) is installed in your airflow environment
+- Create a sample DAG file like [`generic_recipe_sample_dag.py`](../src/datahub_provider/example_dags/generic_recipe_sample_dag.py). This will read your DataHub ingestion recipe file and run it.
```

Comment:
How does the DAG get access to the recipe file when executing in the Airflow executor?

Reply:
added some details